test/index.js

```diff
@@ -57,12 +55,7 @@ function compare (t, name, log) {
     })
 }
 
-function error (name, cb) {
+function error (name) {
   const html = fs.readFileSync(path.join(fixtures, `${name}.html`), 'utf8')
-
-  try {
-    return reshape({ parser }).process(html)
-  } catch (err) {
-    cb(err.toString())
-  }
+  return reshape({ parser }).process(html)
```

Comment:
I think you'll need to upgrade the version in your package.json to get this passing on travis